_component_design/time-picker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Time Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to select a specific time or window from a sequential list. The Time Picker is often accompanied by a date picker for the purpose of creating a date range filter.
+preview-image: preview-images/time-picker.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+Use the time picker if the list of times are sequential, such as 9am, 10am, 11am, and so on.
+
+Do not use the Time Picker if there are gaps between time options. For example, if 9am, 12pm, and 3pm are the options, use radio buttons. 
+
+### Best practices
+
+* Times in a list should be incremental, usually by every 30 minutes or 1 hour, but 10 or 15 minute increments may be used depending on the use case.
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.cdn_url}}/assets/images/components/inputs-and-controls/time-picker/time-picker-hero-image.svg)
```

Comment:
Gotta ask: Is that icon supposed to be a clock with a checkmark or a clock partially obscuring a down caret?

Reply:
it's a clock with a check-mark